skel/flatpak/overrides/global

```diff
@@ -0,0 +1,6 @@
+[Environment]
+QT_LINUX_ACCESSIBILITY_ALWAYS=true
+GTK_PATH=/app/lib/gtkmodules
```

Comment:
This shouldn't be required, the AT-SPI module is built with Gtk itself so the default paths should be enough